src/App/docutils/ComponentPreview/components/Topbar.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
+import { tabs as tabsScript } from "@src/scripts/main";
+
+export const Topbar = ({
+    tabsId,
+    tabs,
+    hideOptions,
+    sidebarDisplayed,
+    setSidebarDisplayed,
+    updateActiveTab,
+    activeTab
+}) => (
+    <div id={tabsId} className="tabs tabs-scroll">
+        { tabs && <Tabs
+            tabsId={tabsId}
+            tabs={tabs}
+            activeTab={activeTab}
+            updateActiveTab={updateActiveTab}
+        />}
+        <CloseOptionsButton
+            hideOptions={hideOptions}
+            sidebarDisplayed={sidebarDisplayed}
+            setSidebarDisplayed={setSidebarDisplayed}
+
+        />
+    </div>
+);
+
+/* TODO: switch anchor links to buttons */
```

Comment:
Shall we just do it perhaps? Or do you think it is better to do it in an iteration?

Reply:
I think this should be done by fixing the Tabs component. Once the Tabs styling support button tabs and not only anchor-link tabs, we will be able to easily switch to button, and everybody else too in their own projects
Jira ticket about it is created, and reminder for the sandbox tabs is added in the iteration jira ticket 👍
comment removed 🧹 boop ! 🌪